src/bundle/Editor.ts

```diff
@@ -127,6 +135,7 @@ export type EditorOptions = Pick<
     preset: EditorPreset;
     directiveSyntax: DirectiveSyntaxContext;
     pmTransformers: TransformFn[];
+    preserveMarkupFormatting?: boolean;
```

Comment:
unnecessary parameter (u can get this value from experimental property)

Reply:
fixed